index.js

```diff
@@ -1,21 +1,18 @@
 document.addEventListener("DOMContentLoaded", () =>{
-	let root = document.documentElement;
+	const root = document.documentElement;
```

Comment:
This can be a `const`, as it's constant, it never changes. You should only use `let` when you expect to change that variable.